docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile

```diff
@@ -0,0 +1,109 @@
+# Dockerfile for libcudacxx_base:host_x86_64_ubuntu_20.04__target_x86_64_ubuntu_20.04__clang_10_cxx_11

```

Comment:
Please fix the comments to refer to Clang 11 and not 10. (Same for the other files.)

Reply:
@griwes 
Thanks Michal!
I'll modify them right now

Reply:
@griwes 
Done

Reply:
For the converting from DOS line endings to Unix line endings, I'll try to fix it now.

Reply:
Done the last task. (submitted by my own Github account)